packages/ui-components/src/components/text-field/text-field.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../assets/styles/globals';
+
+$defaultInputHeight: var(--input-height, 35px);
```

Comment:
keeping our Spatial System in mind, I suggest changing the height to a multiple of 4, in this case 36px.